charts/devlake/values.yaml

```diff
@@ -82,6 +82,8 @@ mysql:
 
   containerSecurityContext: {}
 
+  annotations: {}
+
```

Comment:
kindly change annotations to podAnnotations

Reply:
updated to `podAnnotations` as requested